mmrazor/engine/runner/autoslim_val_loop.py

```diff
@@ -34,15 +37,18 @@ def run(self) -> None:
         all_metrics = dict()
 
         self._model.set_max_subnet()
+        self.calibrate_bn_statistics(self.runner.train_dataloader, 2000)
         metrics = self._evaluate_once()
         all_metrics.update(add_prefix(metrics, 'max_subnet'))
 
         self._model.set_min_subnet()
+        self.calibrate_bn_statistics(self.runner.train_dataloader, 2000)
```

Comment:
pass `self.calibrated_sample_nums` to `calibrate_bn_statistics`

Reply:
Will be fixed in another pr